### DISCUSSION TITLE
How production ready is the 2.4 branch?

### DISCUSSION THREAD

**Discussion by Paktosan** [n0emis/ansible-role-bigbluebutton#286] [Q&A] [answered]
I have seen that there is a 2.4 branch by @BrutalBirdie, is this already production ready?
I have not been following the BigBlueButton development too much but want some features that are available in the new release.

**Comment by Paktosan** [comment, marked as answer]
Since there now is a proper release(https://github.com/ebbba-org/ansible-role-bigbluebutton/releases/tag/v2.4.0), this question has become obsolete.

**Comment by BrutalBirdie** [comment]
I don't know why but I only got a notification now about you solving this discussion.
